src/index.js

```diff
@@ -40,11 +40,14 @@ export default function commonjs(options = {}) {
 			} catch (err) {
 				resolvedId = resolve(id);
 			}
+
+			// ASSERT: resolvedId is a normalized path
```

Comment:
Not sure what to make of this comment and the one below. Are these TODOs?

Reply:
Trying to point out that if the assert isn't true, bad things happen. Right now we can assume resolveId is normalized as both versions of `resolve` return normalized paths. If this changes in the future, things will break. I can just remove the comments.